src/vs-bicep/Bicep.VSLanguageServerClient.IntegrationTests/CompletionTests.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.VSLanguageServerClient.IntegrationTests.Utilities;
+using Microsoft.Test.Apex.VisualStudio.Editor;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Bicep.VSLanguageServerClient.IntegrationTests
+{
+    [TestClass]
+    public class CompletionTests : VisualStudioBicepHostTest
+    {
+        [TestMethod]
+        public void Verify_TopLevelPlainTextDeclarations()
+        {
+            ProjectItemTestExtension projectItem = TestProject!["main.bicep"];
+            IVisualStudioTextEditorTestExtension editor = projectItem.GetDocumentAsTextEditor().Editor;
+
+            // This is a hack to wait for language service activation. If this is the first test, it may
```

Comment:
Suggestion: Instead of having this code and comment in every test, how about move this call and its comment to a new function WaitForVS() or WaitForVSToBeResponsive() or something.  That's what you're really doing anyway, the colorization call is just implementation detail.

Reply:
perfect